pkg/config/generic.go

```diff
@@ -171,6 +171,7 @@ type Container struct {
 
 // TestConfiguration provides generic test related configuration
 type TestConfiguration struct {
+	TurniumBonder       ContainerIdentifier               `yaml:"turniumBonder" json:"turniumBonder"`
```

Comment:
This should be in its own Turnium-specific configuration `struct` and registered with the `ConfigPool`. `generic` config is for the generic test suite.

Reply:
Additionally, exported struct elements should be documented (even though they aren't for others below)...